client/components/plans/plan-list/index.jsx

```diff
@@ -78,6 +79,10 @@ const PlanList = React.createClass( {
 			const filteredPlans = filterPlansBySiteAndProps( plans, site, hideFreePlan, showJetpackFreePlan );
 
 			plansList = filteredPlans.map( plan => {
+				if ( ! isEnabled( 'upgrades/personal-plan' ) && plan.product_slug === 'personal-bundle' ) {
```

Comment:
I kinda feel tha for test purposes it would be better to inject additional plan data in calypso instead of filtering-out in mobile apps.
This is how approached this stuff in WordAds new feature:
https://github.com/Automattic/wp-calypso/pull/5292/files#diff-6a910cd6dba5aef1fc29d44512a1da78R81